spec/javascripts/Navigation/header/DocumentationMenu.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import { DocumentationMenu } from 'Navigation/components/header/DocumentationMenu'
+import { DocumentationMenuItem } from 'Navigation/components/header/DocumentationMenuItem'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+const Props = {
+  docsLink: 'docs-link',
+  isSaas: 'true',
+  docsLinksClass: 'docs-class',
+  customerPortalLink: 'cp-link',
+  apiDocsLink: 'apidocs-link',
+  liquidReferenceLink: 'lr-link',
+  whatIsNewLink: 'ws-new-link'
+}
+
+describe('<DocumentationMenu/>', () => {
+  it('renders <DocumentationMenu/> component when Saas', () => {
+    const wrapper = shallow(<DocumentationMenu {...Props} />)
+    expect(wrapper.find('.PopNavigation--docs')).toHaveLength(1)
+    expect(wrapper.find('a.PopNavigation-trigger')).toHaveLength(1)
+    expect(wrapper.find('ul.PopNavigation-list')).toHaveLength(1)
+    expect(wrapper.find(DocumentationMenuItem)).toHaveLength(4)
+  })
+
+  it('renders <DocumentationMenu/> component when no Saas', () => {
+    let props = Object.assign({}, Props, {isSaas: null})
+    const wrapper = shallow(<DocumentationMenu {...props} />)
+    expect(wrapper.find('.PopNavigation--docs')).toHaveLength(1)
+    expect(wrapper.find('ul.PopNavigation-list')).toHaveLength(1)
+    expect(wrapper.find(DocumentationMenuItem)).toHaveLength(3)
```

Comment:
Perhaps this `3` should be extracted as a constant or described somehow.

Reply:
Yeah, I understand there's an extra Item for SaaS, but would love to have it reflected in the test.